### ISSUE TITLE
youtube loadvideobyid fails the first time

### ISSUE THREAD

**Issue by caranicas** [davatron5000/FitVids.js#222]
When trying to load a new video by id the player replays the the video that the player was initialized with. All subsequent attempts work correctly.

**Comment by davatron5000** [comment]
We don't have a `loadvideobyid` method in our project. Are you sure you're in the right place.

Also we require a [reduced test case](https://css-tricks.com/reduced-test-cases/) to help debug issues.

**Comment by caranicas** [comment]
I'm sorry that I wasn't more clear. I was talking about the youtube iframe method loadvideobyid. When I don't use fitvid it works as expected, it would load and play the new video. However once I integrated fitvid when I tried to load a second video, it would load but it then it would swap back to the video that had been loaded when I initialized the player. However all subsequent loads worked as expected.  

I am at work right now, but I will attempt to work on reduced test case this weekend. thanks.
